.github/workflows/conformance.yml

```diff
@@ -79,7 +79,7 @@ jobs:
         /usr/local/bin/kubectl apply -f ./config/crd/experimental/policy.networking.k8s.io_adminnetworkpolicies.yaml
         /usr/local/bin/kubectl apply -f ./config/crd/experimental/policy.networking.k8s.io_baselineadminnetworkpolicies.yaml
         # install kube-network-policies
-        /usr/local/bin/kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/kube-network-policies/v0.2.0/install-anp.yaml
+        /usr/local/bin/kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/kube-network-policies/main/install-anp.yaml
```

Comment:
installing an unversioned file from knp seems like it will cause problems at some point

Reply:
yeah @aojea will cut a release soon.. waiting for that, this was in the interim to see if master fixed the bug we were seeing..

Reply:
here you go https://raw.githubusercontent.com/kubernetes-sigs/kube-network-policies/refs/tags/v0.6.1/install-anp.yaml

Reply:
@aojea how does kube-network-policies get consumed into KIND? so which version of k-net-pol does kind 0.24 use?

Reply:
ETA for a new release of kind = kubecon 😄

Reply:
@tssurya ben just released a new version of kind , it should work with the new version